webapp/src/components/containers/DropdownCard.vue

```diff
@@ -24,14 +24,26 @@
 </template>
 
 <script setup lang="ts">
-import { ref } from "vue";
+import { ref, onMounted } from "vue";
 
 import UpArrow from "@/assets/svg/UpArrow.vue";
 import DownArrow from "@/assets/svg/DownArrow.vue";
 
 import IconCardHeader from "./IconCardHeader.vue";
 
+const props = defineProps({
+  initiallyOpen: {
+    type: String,
+    default: "true", //open by default
+  },
+});
```

Comment:
I personally find it clearer to use [type-only props declarations](https://vuejs.org/api/sfc-script-setup.html#type-only-props-emit-declarations). also, you can have a preciser type (only two value are possible here, true or false) and avoid `JSON.parse` at all.

```suggestion
const props = withDefaults(
  defineProps<{
    initiallyOpen?: boolean;
  }>(),
  { initiallyOpen: true },
);
```